pkgs/development/tools/misc/elfutils/default.nix

```diff
@@ -41,14 +41,16 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     patchShebangs tests/*.sh
+  '' + lib.optionalString enableDebuginfod ''
```

Comment:
Any chance you could add a comment around here why this is desired? (Bonus points for adding it to the commit message, but IMHO a note around here is much more useful / important.)